src/usr/local/www/widgets/widgets/system_information.widget.php

```diff
@@ -474,7 +477,6 @@ function get_sysinfo_item_html($itemkey) {
 </div><div id="widget-<?=$widgetname?>_panel-footer" class="panel-footer collapse">
 
 <form action="/widgets/widgets/system_information.widget.php" method="post" class="form-horizontal">
-<form action="/widgets/widgets/system_information.widget.php" method="post" class="form-horizontal">
```

Comment:
no, there was a custom title (new widget system) taken out, I probably replaced it not noticing the original line was just above it